h2o-automl/src/main/java/ai/h2o/automl/ModelSelectionStrategies.java

```diff
@@ -38,11 +40,11 @@ public KeepBestN(int N, Supplier<LeaderboardHolder> leaderboardSupplier) {
 
         @Override
         @SuppressWarnings("unchecked")
-        public Selection<M> select(Key<M>[] originalModels, Key<M>[] newModels) {
+        public Selection<M> select(Key<M>[] originalModels, Key<M>[] newModels, EventLog eventLog) {
```

Comment:
no really, now having this eventlog passed everywhere is worse than not having it at all

Reply:
you can now remove this EventLog param, right?

Reply:
Yes I can, thank you!